service/src/main/java/uk/gov/hmcts/reform/fpl/model/Judge.java

```diff
@@ -34,4 +35,12 @@ public String getJudgeName() {
         }
         return judgeLastName;
     }
+
+    @JsonIgnore
+    public boolean equalsJudgeAndLegalAdvisor(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
```

Comment:
I've updated this in my PR so take a look (but as discussed, may need to re-think how to handle judge email addresses)